hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -689,6 +698,8 @@ private synchronized void computeAndUpdateLastAppliedIndex(
         }
       }
     }
+    this.metrics.updateApplyTransactionMapSize(applyTransactionMap.size());
```

Comment:
applyTransactionMap.size() represents pending apply transasctions or transaction-in-progress, can rename metrics as per that

Reply:
I would prefer the current name.